### PR TITLE
fix: focustrap when no buttons are mounted right away

### PR DESCRIPTION
**Pull request by Bilb**

### CODE REVIEW

File: ts/components/SessionFocusTrap.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const DefaultFocusButton = styled.button`
+  opacity: 0;
+  width: 0;
+  height: 0;
+`;
+
+/**
+ * The FocusTrap always require at least one element to be tabbable.
+ * On some dialogs, we don't have any until the content is loaded, and depending on what is loaded, we might not have any tabbable elements in it.
+ * This component renders the children inside a FocusTrap and always adds an invisible button to make FocusTrap happy.
```
Comment (yougotwill) [inline ts/components/SessionFocusTrap.tsx:14]: I have come up with a slightly cleaner solution in my opinion. Have tested on my machine and confirmed working. Can you update your code and test please? We can also should test on windows to be 100% through.

```tx
import FocusTrap from 'focus-trap-react';
import React, { ReactNode, useState } from 'react';
import { useMount } from 'react-use';

type Props = {
  children: ReactNode;
};

export function SessionFocusTrap(props: Props) {
  const [active, setActive] = useState(false);

  useMount(() => setActive(true));

  return (
    <FocusTrap active={active} focusTrapOptions={{ initialFocus: false, allowOutsideClick: true }}>
      {props.children}
    </FocusTrap>
  );
}

```

**Review by yougotwill — CHANGES_REQUESTED** [review]

File: ts/components/SessionFocusTrap.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import FocusTrap from 'focus-trap-react';
+import React, { useState } from 'react';
+import useMount from 'react-use/lib/useMount';
```
Comment (yougotwill) [inline ts/components/SessionFocusTrap.tsx:3]: Can the import not be just 'react-use'?

**Review by yougotwill — CHANGES_REQUESTED** [review]

File: ts/components/SessionFocusTrap.tsx (hunk 2 above)
Comment (Bilb) [reply ts/components/SessionFocusTrap.tsx]: resolved

**Review by yougotwill — APPROVED** [review]

**Comment by Bilb** [comment]
confirmed fine on windows too 
image

**Comment by yougotwill** [comment]
LGTM! 🚢

**Review by yougotwill — APPROVED** [review]
LGTM 🚢

**Pull request merged**